src/v/kafka/server/handlers/incremental_alter_configs.cc

```diff
@@ -389,6 +389,20 @@ create_topic_properties_update(
                   op);
                 continue;
             }
+            if (cfg.name == topic_property_iceberg_target_lag_ms) {
+                parse_and_set_optional(
+                  update.properties.iceberg_target_lag_ms,
+                  cfg.value,
+                  op,
+                  iceberg_target_lag_ms_validator{},
+                  [](const ss::sstring& v) {
```

Comment:
do you need to pass this lambda? we have other "ms" properties and default works fine for them but why not for this?

Reply:
i recall having to add it for this particular overload, but I will double check.

Reply:
https://github.com/redpanda-data/redpanda/blob/e144981e1d0d1c335898a10bef7f7805d384d86e/src/v/kafka/server/handlers/configs/config_utils.h#L645-L650

🎉